public/js/action-list.js

```diff
@@ -106,6 +117,13 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            // If the item exists, remove it first before reading it
+            if (bottomStatusBar.find('.selected-items').length !== 0) {
```

Comment:
Consider processing this as a boolean: `if (seq.length) {` or `if (! seq.length) {`